source/app/static/assets/js/iris/case.notes.js

```diff
@@ -358,6 +362,45 @@ async function note_detail(id) {
     });
 }
 
+function handle_ed_paste(event) {
+    filename = null;
+    const { items } = event.originalEvent.clipboardData;
+    for (let i = 0; i < items.length; i += 1) {
+      const item = items[i]; 
+
+      if (item.kind === 'string') {
+        item.getAsString(function (s){
+            filename = $.trim(s.replace(/\t|\n|\r/g, '')).substring(0, 40);
+        });
+      }
+
+      if (item.kind === 'file') {
+        const blob = item.getAsFile();
+		
+        if (blob !== null) {
+            const reader = new FileReader();
+            reader.onload = (e) => {
+				notify_success('The file is uploading in background. Don\'t leave the page');
+
+                if (filename === null) {
+					filename = random_filename(25);
+                }
+
+                upload_interactive_data(e.target.result, filename, function(data){
+                    url = data.data.file_url + case_param();
+                    event.preventDefault();
+                    editor.insertSnippet(`\n![${filename}](${url} =40%x40%)\n`);
```

Comment:
```suggestion
                    note_editor.insertSnippet(`\n![${filename}](${url} =100%x40%)\n`);
```
Thanks ! We should use the already note_editor